salt/_pillar/metalk8s_private.py

```diff
@@ -1,6 +1,7 @@
 import os.path
 import logging
 
+import pillar_utils
```

Comment:
For me you don't need this and you should not use this.

You should call utils using `__utils__` like
`__utils__['update_with_errors'](blabla)`